pkg/pods/pods.go

```diff
@@ -75,21 +83,23 @@ func RemovePodByManifestPath(manifestPath string) {
 	RemovePod(podToDelete.Name)
 }
 
-func FindFirstManifestPath() string {
+func FindAllManifestPaths() []string {
 	podsPath := context.CurrentContext().PodsDir()
-	files, err := ioutil.ReadDir(podsPath)
+	files, err := os.ReadDir(podsPath)
 	if err != nil {
 		log.Fatal(err.Error())
 	}
 
+	manifestPaths := make([]string, 0)
```

Comment:
make is not required here because append will create it.
```suggestion
	var manifestPaths []string
```